adapters/httpapi_compact.c

```diff
@@ -254,6 +255,7 @@ HTTP_HANDLE HTTPAPI_CreateConnection(const char* hostName)
                 http_instance->certificate = NULL;
                 http_instance->x509ClientCertificate = NULL;
                 http_instance->x509ClientPrivateKey = NULL;
+                http_instance->tls_renegotiation = 0;
```

Comment:
Shall be `false`. Please see comment below.

Reply:
Fixed on the commit 98666e9b4d46e63d900d4abd1bc1591d3049b5a6